AdNetworkSupport/MoPubMediationLogger.h

```diff
@@ -0,0 +1,61 @@
+//
+// Created by Pooja Shashidhar on 4/16/18.
+// Copyright (c) 2018 MoPub. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface MoPubMediationLogger : NSObject
+
+typedef enum {
+    AdColony,
+    AppLovin,
+    AdMob,
+    Chartboost,
+    FacebookAudienceNetwork,
+    Flurry,
+    IronSource,
+    OneByAOL,
+    Tapjoy,
+    UnityAds,
+    Vungle
+} Network;
```

Comment:
Rename the enum to `AdNetwork` to be consistent with `AdFormat`.

Reply:
Updated